velox/common/base/BitUtil.h

```diff
@@ -958,6 +958,21 @@ inline void padToAlignment(
   }
 }
 
+#if defined __has_builtin
+#if __has_builtin(__builtin_bswap128)
+#define HAS_BUILTIN_BSWAP_INT128 1
+inline __int128_t builtin_bswap128(__int128_t value) {
+  return __builtin_bswap128(value);
+}
+#endif
+#endif
+#if not HAS_BUILTIN_BSWAP_INT128
```

Comment:
maybe add VELOX_ prefix to reduce the chance of collision with some other library

let's also #undef this when no longer needed